pkg/server/api/bundle/v1/service.go

```diff
@@ -0,0 +1,165 @@
+package bundle
+
+import (
+	"context"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/spire/pkg/server/api/rpccontext"
+	"github.com/spiffe/spire/pkg/server/plugin/datastore"
+	"github.com/spiffe/spire/proto/spire-next/api/server/bundle/v1"
+	"github.com/spiffe/spire/proto/spire-next/types"
+	"github.com/spiffe/spire/proto/spire/common"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var defaultMask = &types.BundleMask{
```

Comment:
I think we're going to need/want some common way of applying masks. I think we could wire up a little code generation to the makefile to generate mask application code for the various types. Either way I think this is ok for now.